src/back/TicketManager/tests/TicketManager.IntegrationTests/TestBase.cs

```diff
@@ -83,29 +69,77 @@ public TestBase()
             PhoneNumber = DefaultOrganizer.PhoneNumber,
             TaxId = DefaultOrganizer.TaxId,
             TaxIdType = TaxIdTypeDto.Pesel,
-        }).Wait();
-        // Authorize organizer to be able to login
+        });
         
+        var organizers = await AdminClient.GetSuccessAsync<OrganizerListEndpoint, OrganizerListRequest, PaginatedResponse<OrganizerDto>>(new()
+        {
```

Comment:
Same here, I don't know why C# compiles it. Default values?

Reply:
Yea, default values. `0` for `enum` and `false` for `bool`.

Reply:
BTW the same happens during deserialization if client doesn't specify those fields.